shared/team-building/service-tab-bar.js

```diff
@@ -74,44 +75,33 @@ const ServiceTabBar = (props: Props) => (
 )
 
 const styles = Styles.styleSheetCreate({
-  container: Styles.platformStyles({
```

Comment:
did a tool tell u this was not in use or just eyeballed it?

Reply:
I used `rg`. There are no uses of `styles.container` in this file.